src/gstairplaysrc.c

```diff
@@ -346,6 +353,12 @@ gst_airplay_src_set_property (GObject * object,
   GST_DEBUG_OBJECT (self, "set property");
 
   switch (prop_id) {
+    case PROP_DNSSD_NAME:
+      g_mutex_lock(&self->data->property_lock);
+      g_free (self->data->dnssd_name);
+      self->data->dnssd_name = g_strdup (g_value_get_string (value));
```

Comment:
`g_value_dup_string()`

Reply:
Is this a style or correctness fix?

Reply:
Style, I guess. `g_value_dup_string()` replaces `g_strdup()` and `g_value_get_string()` into one.